OpenStack-accounting/usr/local/sbin/glance_extract_accounting.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+import accountinglib
+
+import sys
+import time
+import datetime
+
+
+nowtime = time.localtime()
+
+print("Glance Accounting run start")
```

Comment:
These print statements should really be logs: i.e. `logger.debug()` 
https://www.geeksforgeeks.org/logging-in-python/